tools/generate-helpers/pg-table-bindings/store.go.tpl

```diff
@@ -353,7 +365,7 @@ func (s *storeImpl) upsert(ctx context.Context, objs ...*{{.Type}}) error {
 }
 {{- end }}
 
-//// Helper functions - END
+// endregion
```

Comment:
Is it possible (and does it make sense) to enrich this comment with a reference to the region name (Helper functions) ? I think it would improve the readability for non-IDE readers